zipkin-lens/package.json

```diff
@@ -30,11 +30,16 @@
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "@types/fetch-mock": "^7.3.2",
     "@types/jest": "^24.9.1",
+    "@types/lingui__macro": "^2.7.3",
```

Comment:
`lingui` already mostly provides typescript definitions but does have some bugs it seems (fixed at master until they release)

Can you check whether we can add definitions here instead of using these old types?

https://github.com/openzipkin/zipkin/blob/master/zipkin-lens/src/types.d.ts#L16

Reply:
Ah, found linguis `3.0.0-10` is out, let's try updating I think it'll fix the types.

Reply:
Nice!
But I'd like to work on that after this PR, as the major version update may possibly need some major changes for migration.
So for now, I'm going to copy and paste the type definition into type.d.ts.